go-controller/pkg/metrics/metrics.go

```diff
@@ -411,3 +411,18 @@ func RegisterOvnMetrics(clientset kubernetes.Interface, k8sNodeName string) {
 	go RegisterOvnControllerMetrics()
 	go RegisterOvnNorthdMetrics(clientset, k8sNodeName)
 }
+
+// StartOVSMetricsServer runs the prometheus listener so that OVS metrics can be collected
+func StartOVSMetricsServer(bindAddress string) {
+	handler := promhttp.InstrumentMetricHandler(ovnRegistry,
+		promhttp.HandlerFor(ovnRegistry, promhttp.HandlerOpts{}))
```

Comment:
@martinkennelly ah good point. Does that mean I should skip the OVS specific bind address, and just make it a boolean to enable OVS metrics since they'll be handled by the default metrics handler?

Reply:
Yeah, boolean makes sense.
Id like the OVS metrics to be exposed at the OVN endpoint and with the ovnRegistry.
Therefore, the OVS metrics need to be refactored and depending on the context, either register with ovnRegistry or default registry (to accommodate existing metrics exporter app).

Reply:
@dcbw ^

Reply:
Done!